README.md

```diff
@@ -146,3 +146,21 @@ The output is written to `Filters/filter.txt`.
 ### Others
 
 * [Additional rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/rules.txt)
+
+## AdGuard DNS filter for popup domains
+
+The filter is designed to provide a blocking page address in response to popup domain requests. For this purpose the modifiers all, popup and document are replaced by the modifier dnsrewrite with the value of blocking page.
+
+## AdGuard DNS filter for popup domains structure
```

Comment:
We should explain here that technically we're only using rules with `$popup`, `$document` and `$all` modifiers as this effectively means that these rules target websites that are opened in the main frame.